.travis.yml

```diff
@@ -31,7 +31,7 @@ script:
 
 env:
   matrix:
-    - PYTHONASYNCIODEBUG=1 AIOHTTP_NO_EXTENSIONS=0
-    - PYTHONASYNCIODEBUG=0 AIOHTTP_NO_EXTENSIONS=0
-    - PYTHONASYNCIODEBUG=1 AIOHTTP_NO_EXTENSIONS=1
-    - PYTHONASYNCIODEBUG=0 AIOHTTP_NO_EXTENSIONS=1
+    - PYTHONASYNCIODEBUG=nonemptystring AIOHTTP_NO_EXTENSIONS=
```

Comment:
It affects aiohttp contributors only.
Believe me I'll never again forget the rule but I don't want to see `nonemptystring` every time when I open `.travis.yml`.